### ISSUE TITLE
versions 0.2.x doesn't work with coffeelint-no-implicit-returns

### ISSUE THREAD

**Issue by jellymann** [AtomLinter/linter-coffeelint#35]
Using Atom version 0.201 and linter-coffeelint@0.2.3 on OSX 10.10.2 with coffeelint-no-implicit-returns yields the following error:

```
Cannot find module 'coffeelint-no-implicit-returns' from 'file'
/Users/<username>/.atom/packages/linter-coffeelint/lib/linter-coffeelint.coffee:61 Error: Cannot find module 'coffeelint-no-implicit-returns' from 'file'
  at module.exports (/Users/<username>/.atom/packages/linter-coffeelint/node_modules/coffeelint/node_modules/resolve/lib/sync.js:32:11)
  at expandModuleNames (/Users/<username>/.atom/packages/linter-coffeelint/node_modules/coffeelint/lib/configfinder.js:78:35)
  at Object.exports.getConfig (/Users/<username>/.atom/packages/linter-coffeelint/node_modules/coffeelint/lib/configfinder.js:111:16)
  at LinterCoffeelint.lintFile (/Users/<username>/.atom/packages/linter-coffeelint/lib/linter-coffeelint.coffee:49:29)
  at /Users/<username>/.atom/packages/linter/lib/linter-view.coffee:163:18
  at Array.forEach (native)
  at /Users/<username>/.atom/packages/linter/lib/linter-view.coffee:162:18
  at FSReqWrap.oncomplete (fs.js:77:15)
```

Workaround is downgrading to linter-coffeelint@0.1.8

**Comment by Gerhardk** [comment]
I have the same issue!

**Comment by AsaAyers** [comment]
I need to take some time to add a warning if you're using 3rd party plugins with a version of CoffeeLint that doesn't support it

> I fixed this in CoffeLint v1.9.5. If your project uses a local coffeelint (devDependency in your package.json) you will need to update that and upgrade linter-coffeelint to v0.2.3.
> 
> linter-coffeelint < 0.2.3 has a bug that causes it to not be able to find the local coffeelint 7ffb6eb.

https://github.com/AtomLinter/linter-coffeelint/issues/25#issuecomment-99122300

**Comment by AsaAyers** [comment]
The other possibility that I know if is that you forgot to install `coffeelint-no-implicit-returns` locally. It should be in the `devDependencies` of your project.
